sdk/dotnet/ApiRegistration/V1/APIService.cs

```diff
@@ -54,12 +54,12 @@ public partial class APIService : KubernetesResource
         /// <param name="args">The arguments used to populate this resource's properties</param>
         /// <param name="options">A bag of options that control this resource's behavior</param>
         public APIService(string name, Types.Inputs.ApiRegistration.V1.APIServiceArgs? args = null, CustomResourceOptions? options = null)
-            : base("kubernetes:apiregistration/v1:APIService", name, SetAPIKindAndVersion(args), options)
+            : base("kubernetes:apiregistration.k8s.io/v1:APIService", name, SetAPIKindAndVersion(args), options)
```

Comment:
Looks like .NET codegen isn't generating aliases

Reply:
Opened #1022 to track that.